examples/00-EDB/10_GDS_workflow.py

```diff
@@ -0,0 +1,104 @@
+"""
+EDB: Edit Control File and import gds
+-------------------------------------
+This example shows how you can use PyAEDT to import a gds from an IC file.
```

Comment:
```suggestion
EDB: Edit control file and import GDS
----------------------------------------
This example shows how you can use PyAEDT to import a GDS from an IC file.
```
@maxcapodi78 What does GDS stand for? I think we need to define it. Is it Generalized Desired State? I probably incorrectly edited your other PR believing it to be a file format.